File: frontend/src/components/DaySchedule.js
import React from "react";

export const DAY_SCHEDULE = [
  ["NONE", "(no information)"],
  ["INOFFICE", "In Office"],
  ["WFH", "Work From Home"],
  ["OFF", "Off"]
];

export default ({ daySchedule }) => (
  <React.Fragment>
    {daySchedule && DAY_SCHEDULE.find(p => p[0] === daySchedule || "NONE")[1]}
  </React.Fragment>
);

File: backend/atlas/schema/dayschedule.py
import graphene


class DaySchedule(graphene.Enum):
    WFH = "WFH"
    INOFFICE = "INOFFICE"
    OFF = "OFF"
    NONE = ""

    @property
    def description(self):
        if self == DaySchedule.WFH:
            return "Work From Home"
        if self == DaySchedule.INOFFICE:
            return "In Office"
        if self == DaySchedule.OFF:
            return "Off"
        return ""
